### ISSUE TITLE
Wrong regex for timestamp parser

### ISSUE THREAD

**Issue by glenncoppens** [nielsbasjes/logparser#22]
Hi 

First of all, tnx for the great library.

But when I used the logparser in java, using an old log file from 1995 I couldn't get the parsing to work properly.
After long time debugging and trying I finally found that the year in a request's timestamp has to begin with a 2. It's hardcoded in the timestamp regex.
```
parsers.add(new TokenParser("%t", "request.receive.time", "TIME.STAMP", Casts.STRING_ONLY, "\\[[0-3][0-9]/(?:[A-Z][a-z][a-z])/2[0-9][0-9][0-9]:[0-9][0-9]:[0-9][0-9]:[0-9][0-9] [\\+|\\-][0-9][0-9][0-9]0\\]"));
```

Maybe there's a reason why you did this? 
Otherwise it would be cool to make it work for less than the year 2000 ;)

Glenn

**Comment by nielsbasjes** [comment]
Hi, Thanks for letting me know you appreciate the tool!
There is no real reason for only allowing years in the 2000-2999 range.
At the time I wrote it this seemed like a fine range because I figured "no one will want to parse logfiles more than 13 years old" .... Ok, I was wrong about that.
I'll fix it.

**Comment by nielsbasjes** [comment]
I committed the fix to trunk: https://github.com/nielsbasjes/logparser/commit/3895e262745b7ad3e4d9c3a06080cb5cd2c4fc80

Either build the trunk (which should work), apply this to the 1.9.1 and build your own version or wait till I release the next version (which will take some time!!)

**Comment by nielsbasjes** [comment]
Hi @glenncoppens, I just released 2.0 which contains this fix.
